test/integration/good/pretty.expected

```diff
@@ -1157,221 +1157,355 @@ data {
   int a0;
   int a1[1];
   int a2[2, 3];
+  int a3[1];
+  int a4[2, 3];
   real b0;
   real b1[4];
   real b2[5, 6];
+  real b3[4];
+  real b4[5, 6];
   vector[7] c0;
   vector[7] c1[8];
   vector[7] c2[8, 9];
+  vector[7] c3[8];
+  vector[7] c4[8, 9];
```

Comment:
So the pretty printer reverts code to the old syntax. That seems odd; isn't the new syntax meant to be a better replacement for the old? If so, the printer should output should reflect that.

Or maybe the AST should remember which syntax was used and pretty printer preserves it. I'd like to keep syntax information in the AST. The canonicalizer can change it to the "preferred" form.

Reply:
Agreed. My plan was to make that change in a future PR, but I could easily add it if that's better

Reply:
The pretty printer can wait. I think the AST change would belong to this PR.

Reply:
Is the AST currently sensitive to style anywhere? It currently throws out e.g. whitespace and comments. IMO, preserving code style is a pretty significant change in the goal of the AST, and this would be the only instance AFAIK. Do you have an important use case in mind?

Reply:
I don't know but [here](https://github.com/stan-dev/stanc3/issues/65#issuecomment-468362126) Matthis says the current AST is "full fidelity" and people discuss why that is a good thing. Throwing out comments is a [mistake](https://github.com/stan-dev/stanc3/issues/93). I consider it a critical bug that precludes the use of `--auto-format`.

Reply:
I see what you're saying; you're considering it a bug that e.g. comments are thrown out, and this would make that bug worse by not including this element of style in the AST.

But I was misleading, this isn't a feature of style like whitespace or comments; we're planning on depreciating the old syntax, so really we want to always print out the new syntax. When we do that we won't need this information in the AST, it would become obsolete once the pretty printer changes are made, right?

Reply:
The problem with omitting comments is that `--auto-format` doesn't improve the readbility of a complex model--without comments it's even worse. I don't care about other syntax changes.

Note that  `GetLP` and `GetTarget` are exactly the same except that one is deprecated. The current AST considers them separate anyway. The pretty printer prints the AST faithfully; deprecated constructs are cleaned up by the canonicalizer. This division of labor makes sense to me.

Reply:
Good point about the precedent with `GetLP` and `GetTarget`, especially because it was necessary to parse them differently in order to emit a depreciation warning. I agree it's necessary to put them in the AST for that reason at minimum. I'll add that!